src/main/java/uk/gov/pay/webhooks/message/dao/entity/WebhookMessageEntity.java

```diff
@@ -30,27 +30,27 @@
 
 @NamedQuery(
         name = WebhookMessageEntity.MESSAGE_BY_WEBHOOK_ID_AND_MESSAGE_ID,
-        query = "select m from WebhookMessageEntity m where webhookEntity.externalId = :webhookId and externalId = :messageId"
+        query = "select m from WebhookMessageEntity m where webhookEntity = :webhook and externalId = :messageId"
```

Comment:
Is it possible to use condition `webhookEntity.id.= :id` for all queries so it is clear about the field being queried? we are passing entire Webhooks entity to these named queries which may not be needed.

Reply:
I'm not sure if JPA will assume it needs to cross join to get the entity in order to compare an attribute of it, I could try. 

Using this method I don't need to add an accessor for the internal identifier on webhooks and just let JPA figure out its the `@ID` field that makes it unique.

I have checked the output queries just to be sure we're OK from a performance perspective and it does behave as expected (without a cross join, webhook entity marked as `FetchType.LAZY`)

Example from debugger
```
/* WebhookMessage.count_messages_by_webhook_id */ 
select count(webhookmes0_.id) as col_0_0_ from webhook_messages webhookmes0_ where webhookmes0_.webhook_id=$1
```